### ISSUE TITLE
Automation with Ansible?

### ISSUE THREAD

**Issue by z0ph** [sb2nov/mac-setup#222]
### I'm Submitting a ...
```
[ ] Bug report
[x] Tool/language/etc documentation request
```
Hi,

Are you interested if I propose a pull request with a new category trying to automate the deployment of this mac-setup guide with Ansible ? Like this one : https://github.com/z0ph/ansible-mac-install (fork)

zoph.

**Comment by sb2nov** [comment]
Since your work already exists, I'm not sure if there is value in duplicating the effort since your repo should be the source of truth for that.

**Comment by Kyslik** [comment]
@sb2nov I think @z0ph goes for curated Ansible playbooks by authors of mac-setup, since the fork does not 100% reflect whats written in the guide.

**Comment by z0ph** [comment]
Yeah, the idea is to propose a full automation setup of `mac-setup` :)

**Comment by sb2nov** [comment]
Ah might have missed that. Let's go for it then 👍  

Thanks for taking this on.
